tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -34,6 +34,10 @@ workdir="${WORKDIR:-$PWD}"
 
 destdir="${workdir}/kata-static"
 
+KATA_INSTALL_OWNER="${KATA_INSTALL_OWNER:-root}"
```

Comment:
Makes me sad see that block of initialization repeated in many places. is it a bad idea to move them all into a `common.bash` file in the root of the project?